app/controllers/sessions_controller.rb

```diff
@@ -21,4 +18,14 @@ def destroy
     log_out if logged_in?
     redirect_to root_url
   end
+
+  private
+
+  def checkbox_checked? user
+    if params[:session][:remember_me] == Settings.checkbox_checked
```

Comment:
Sửa lại rubocop cho độ dài max là 120 rồi dùng toán tử 3 ngôi đi em

Reply:
done